examples/applications/object_detection/retina_net/basic/pascal_voc/loss.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+
+class FocalLoss(tf.losses.Loss):
+    def __init__(self, num_classes, alpha=0.25, gamma=2.0, delta=1.0, **kwargs):
+        super().__init__(**kwargs)
+        self.num_classes = num_classes
+        self.classification_loss = RetinaNetClassificationLoss(
+            alpha=alpha,
+            gamma=gamma,
+        )
+        self.box_loss = RetinaNetBoxLoss(delta=delta)
+
+    def call(self, y_true, y_pred):
```

Comment:
what's the expect input tensor here? might need some validation if there are certain assumptions here.

Reply:
https://github.com/keras-team/keras-cv/pull/615